server/src/main/java/com/objectcomputing/checkins/services/file/FileServicesImpl.java

```diff
@@ -197,11 +203,16 @@ public FileServicesImpl(GoogleDriveAccessor googleDriveAccessor, EmailSender ema
             CheckinDocument cd = new CheckinDocument(checkInID, uploadedFile.getId());
             checkinDocumentServices.save(cd);
 
-            emailSender.sendEmail("New Check-in Notes", "New check-in notes have been uploaded. Please check the Google Drive folder.");
+//            emailSender.sendEmail("New Check-in Notes", "New check-in notes have been uploaded. Please check the Google Drive folder.");
```

Comment:
can this commented out code be removed?

Reply:
This was part of the old uploadController. It was sending email to hr@objectcomputing.com from kimberlinm@objectcomputing.com after every successful file upload. Since the new file service did not have an AC for sending email after file upload, I've commented it out. (I am assuming that we will add this back at some point)

The logic for this module is available at java/com/objectcomputing/checkins/notifications/email